src/models/order_rewards_schema.py

```diff
@@ -35,6 +35,7 @@ def from_pdf_to_dune_records(cls, rewards_df: DataFrame) -> list[dict[str, Any]]
                     "quote_buy_amount": str(row["quote_buy_amount"]),
                     "quote_gas_cost": float(row["quote_gas_cost"]),
                     "quote_sell_token_price": float(row["quote_sell_token_price"]),
+                    "protocol_fee_recipient": row["protocol_fee_recipient"],
```

Comment:
I would probably have called it `integrator_fee_recipient` to keep the distinction between protocol fees and integrator fees in the outwards communication. The fact that we internally use the same infrastructure is more of an implementation detail.